### PR TITLE
[idea] #297 Opt-Out mode for plugin

### PR DESCRIPTION
**Pull request by keerath**
User Agent and Beagle ID is not tracked if a user opts from this mode

### CODE REVIEW

File: plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/ElasticSearchPanel.java [hunk 1]
```diff
@@ -36,34 +36,30 @@
     private static final String OVERRIDE = "Override";
 
     private static final int[] ELASTIC_SEARCH_PANEL_COLUMN_WIDTHS =
-            new int[] {0, 0, 0, 0, 0, 0, 285, 95, 0};
+            new int[] {29, 84, 63, 285, 0, 95, 0};
```
Comment (keerath) [inline plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/ElasticSearchPanel.java]: Only column dimensions changed in this file..!!

File: plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/ImportsPanel.java [hunk 2]
```diff
@@ -33,7 +33,7 @@
     private static final String EXCLUDE_IMPORTS = "Exclude the following imports:";
 
     private static final int[] IMPORTS_PANEL_COLUMN_WIDTHS =
-            new int[] {0, 48, 0, 91, 218, 53, 55, 17, 0, 0};
+            new int[] {29, 48, 0, 91, 218, 53, 55, 17, 24, 0};
```
Comment (keerath) [inline plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/ImportsPanel.java]: Column dimensions changed

File: plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/LimitsPanel.java [hunk 3]
```diff
@@ -51,13 +51,13 @@
     public static final int TOP_COUNT_MAX = 20;
 
     private static final int[] LIMITS_PANEL_COLUMN_WIDTHS =
-            new int[] {0, 0, 0, 0, 0, 37, 25, 41, 0, 0, 0, 0, 0};
+            new int[] {29, 0, 0, 0, 0, 0, 37, 25, 41, 0, 0, 0, 0, 0};
```
Comment (keerath) [inline plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/LimitsPanel.java]: Only Column dimensions changed

File: resources/TestPlan.md [hunk 4]
```diff
@@ -267,4 +267,16 @@ persisted. Thus a user can see the history of all his applied URLs.<br>
 #### 39. [idea] #299 New settings panel fixed for loading values persisted by old settings panel
 ##### fixes <a href='https://github.com/Imaginea/KodeBeagle/issues/299'> #299 </a>
 1. New settings panel will load values persisted by old settings panel by comparing it with their limits. If the value
-lies in the limit then it will be loaded or else default values will be loaded.
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (keerath) [inline resources/TestPlan.md]: @ScrapCodes please delete this while merging

Comment (ScrapCodes) [inline resources/TestPlan.md]: You should delete it.

**Pull request closed without merging**